includes/admin/auth-service-actions.php

```diff
@@ -45,6 +45,7 @@ public function __construct()
 		$post_type = get_post_type();
 		if($post_type = 'calendar'){
 			add_action('admin_init', [$this, 'oauth_check_iftoken_expired']);
+			add_action('admin_init', [$this, 'auth_get_calendarlist']);
```

Comment:
This should not be needed here, but where we need to display list

Reply:
removed